os/arch/arm/src/armv7-a/arm_fullcontextrestore.c

```diff
@@ -62,5 +62,5 @@
 
 void arm_fullcontextrestore(uint32_t *restoreregs)
 {
-  sys_call1(SYS_restore_context, (uintptr_t)restoreregs);
+	sys_call1(SYS_restore_context, (uintptr_t)restoreregs);
```

Comment:
I dont this this alignment change is related to warning.. lets remove this file.